samples/tfm/provisioning_image/README.rst

```diff
@@ -1,7 +1,7 @@
 .. _provisioning_image:
 
-Provisioning image
-##################
+TF-M: Provisioning image
```

Comment:
I know these prefix updates are not directly related to the other update, it was just convenient to use the same branch to fix when it was pointed out by the release team.

Reply:
That's fine, you could use a separate commit to show that they are not related (not important).

Reply:
Yes, that would have been a better way of doing it... :)
Ok, I'll keep it the way it is this time, but will keep in mind having separate commits next time I do something similar.

Reply:
Never mind, I was misreading it!